get-started/the-stack.md

```diff
@@ -10,92 +11,86 @@ products:
+**{{agent}}** is the core ingestion method for security data. As a single, unified agent, it's purpose-built to collect the rich data needed for modern threat detection and response, including:
 
-#### {{beats}} [stack-components-beats]
+* **Endpoint Security:** Collects detailed event data for threat prevention, detection (EDR), and response directly from your endpoints.  
+* **System & Audit Logs:** Gathers security-relevant logs and audit trails from hosts across your environment.  
+* **Network Activity:** Captures network data to help detect intrusions and suspicious behavior.
 
-{{beats}} are data shippers that you install as agents on your servers to send operational data to {{es}}. {{beats}} are available for many standard observability data scenarios, including audit data, log files and journals, cloud data, availability, metrics, network traffic, and Windows event logs. [Learn more about {{beats}}](beats://reference/index.md).
+Fleets of Elastic Agents are managed centrally, simplifying deployment and policy enforcement across thousands of hosts.
 
-#### {{es}} ingest pipelines [stack-components-ingest-pipelines]
+### Store your data
 
-Ingest pipelines let you perform common transformations on your data before indexing them into {{es}}. You can configure one or more "processor" tasks to run sequentially, making specific changes to your documents before storing them in {{es}}. [Learn more about ingest pipelines](/manage-data/ingest/transform-enrich/ingest-pipelines.md).
+{{es}} is the heart of the Elastic Stack, functioning as the central place to store and search your data. It stores data as **JSON documents**, which are structured data objects. These documents are organized into **indices**, which you can think of as collections of similar documents.
 
-#### {{ls}} [stack-components-logstash]
+Elasticsearch is built to be a resilient and scalable distributed system. It runs as a **cluster** of one or more servers, called **nodes**. When you add data to an index, it's divided into pieces called **shards**, which are spread across the various nodes in the cluster. This architecture allows Elasticsearch to handle large volumes of data and ensures that your data remains available even if a node fails.
 
-{{ls}} is a data collection engine with real-time pipelining capabilities. It can dynamically unify data from disparate sources and normalize the data into destinations of your choice. {{ls}} supports a broad array of input, filter, and output plugins, with many native codecs further simplifying the ingestion process. [Learn more about {{ls}}](logstash://reference/index.md).
+Learn more in [The {{es}} data store](/manage-data/data-store.md)
 
+### Visualize and query your data [kibana-navigation-search]
 
-### Store [_store]
+While {{es}} stores your data, **Kibana** is the user interface where you can explore, visualize, and manage it. It provides a window into your data, allowing you to quickly gain insights and understand trends.
```

Comment:
kibana is not the only entrypoint for querying data.

unfortunately, we are 1/10 of the way through a reorganization of the [explore and analyze section](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2795/explore-analyze/), and our first step was retitling the section. however, 90% of querying and analysis is done through elasticsearch API endpoints, and kibana sits on top of it and just allows you to visualize it.

kibana also acts as the UI to all of the other functionality of elasticsearch (not just solutions). you can view them as the frontend and backend of a single product. most things you can do in elasticsearch, you have some (possibly underpowered) way of doing in kibana. it also allows you to do all sorts of admin tasks like managing users.

Reply:
+CC @lcawl